feature/experimental/policy/policy_vrf_selection/otg_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go

```diff
@@ -332,6 +344,7 @@ func testTrafficFlows(t *testing.T, args *testArgs, expectPass bool, flows ...go
 	args.ate.OTG().StartProtocols(t)
 
 	t.Logf("*** Starting traffic ...")
+	waitOTGARPEntry(t)
```

Comment:
Can use the existing function in otgutils instead. Example at :https://github.com/openconfig/featureprofiles/blob/bca2afa2a567c170b9b25e92acfa5e16c222cfd1/feature/experimental/backup_nhg/otg_tests/backup_nhg_test/backup_nhg_test.go#L251

Reply:
I would also move this right after StartProtocols/before the start traffic log entry, the logging will be more clear that way.

Reply:
updated with existing fucntion.